easybuild/easyconfigs/o/orjson/orjson-3.9.15-GCC-12.3.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'orjson'
+version = '3.9.15'
+
+homepage = 'https://github.com/ijl/orjson'
+description = """Fast, correct Python JSON library supporting dataclasses, datetimes, and numpy."""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
```

Comment:
I'm wondering if this couldn't'/shouldn't be dropped to GCCcore, or is there a particular reason why you used GCC?